packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -183,7 +182,10 @@ const main = async () => {
     const appSSRjs = fse.pathExistsSync(appSSRpath)
     const overrideSSRpath = p.join(
         process.cwd(),
-        projectPkg?.mobify?.overridesDir ?? '',
+        typeof projectPkg?.ccExtensibility?.overridesDir === 'string' &&
+            !projectPkg?.ccExtensibility?.overridesDir?.startsWith('/')
+            ? '/' + projectPkg?.ccExtensibility?.overridesDir
+            : projectPkg?.ccExtensibility?.overridesDir ?? '',
```

Comment:
This conditional assignment doesn't read that well. We are checking type, does that mean that is can be a non-string? Will it error out when someone puts an array or object as a value?

There are 2 things we can do:

1. add proper schema validation for the ccExtensibility key in the package.json that will allow use to define the types for the various values, or validation functions for more complex props like this one. 
2. We drop the configurability of the extensibility folder until we do the above.